Tests/VaporFluentTests/CacheTests.swift

```diff
@@ -0,0 +1,52 @@
+import XCTest
+@testable import VaporFluent
+
+class CacheTests: XCTestCase {
+    func testHappyPath() throws {
+        // config specifying memory database
+        var config = Config([:])
+        try config.set("fluent.driver", "memory")
```

Comment:
That could be nice for writing tests, but I don't think many users will be creating Config structs in this way.